modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-creating-azure-bootstrap_{context}"]
+= Creating the bootstrap machine in Azure
+
+You must create the bootstrap machine in Microsoft Azure to use during
+{product-title} cluster initialization. One way to create this machine is to
+modify the provided Azure Resource Manager (ARM) template.
+
+[NOTE]
+====
+If you do not use the provided ARM template to create your bootstrap machine,
+you must review the provided information and manually create the infrastructure.
+If your cluster does not initialize correctly, you might have to contact Red Hat
+support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an Azure account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VNet and associated subnets in Azure.
+* Create and configure networking and load balancers in Azure.
+* Create control plane and compute roles.
+
+.Procedure
+
+. Copy the template from the *ARM template for the bootstrap machine* section of
+this topic and save it as `04_bootstrap.json` on your computer. This template
+describes the bootstrap machine that your cluster requires.
+
+. Export the following variables required by the bootstrap machine deployment:
++
+----
+export BOOTSTRAP_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key <account_key> -c "files" -n "bootstrap.ign" -o tsv`
```

Comment:
This looks like a typo. My understanding is that "<account_key>" should be replaced with the variable ${ACCOUNT_KEY}

Reply:
Got it; I'll update that.